Test/naming-z/README

```diff
@@ -0,0 +1 @@
+## This is a test for issue #622
```

Comment:
Thanks for guarding the fix against future changes by adding these tests. However, rather than creating a new test folder (especially one that will hold just one test), please move this test into the Test/git-issues folder and somehow add `622` to the filename `d-e$f.dfy`.